lib/driver.js

```diff
@@ -497,6 +508,10 @@ class IosDriver extends BaseDriver {
       }
       logger.debug('Waiting for initial webview');
       await this.navToInitialWebview();
+      // we don't need UIAutoClient and Instruments any more,
+      // should stop them to avoid 'Abnormal Instruments error' due to the command timeout.
+      await this.stopUIAutoClient();
```

Comment:
Hi @jlipps 

I see we got failed when running with CI.
But "dbug Instruments [INST] 2016-09-15 15:18:19 +0000 Debug: Unlock failed. Retrying up to 1 more time(s)." seem doesn't relate with my code. So now I would like to run it in my local. How can I do!